Table.html

```diff
@@ -0,0 +1,246 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Table</title>
+    <style>
+        body{
+            font-family: sans-serif;
+            background-color: aliceblue;
+        }
+        section {
+            position: relative;
+            border: 1px solid #cac6c6;
+            padding-top: 37px;
+            background: #82dede;
+            margin-left: 100px;
```

Comment:
You can use the shorthand property for the margin property. Margin: 0 100px;

Reply:
Done